Shared/SentryIntegration.swift

```diff
@@ -40,6 +43,21 @@ public class SentryIntegration {
             Client.shared = try Client(dsn: dsn)
             try Client.shared?.startCrashHandler()
             enabled = true
+
+            // If we have not already for this install, generate a completely random identifier
+            // for this device. It is stored in the app group so that the same value will
+            // be used for both the main application and the app extensions.
+            if let defaults = UserDefaults(suiteName: AppInfo.sharedContainerIdentifier), defaults.string(forKey: SentryDeviceAppHashKey) == nil {
```

Comment:
I wonder if we should use `NSUserDefaultsPrefs(prefix: "sentry")` for consistency? I guess internally it calls `UserDefaults(suiteName: AppInfo.sharedContainerIdentifier)` anyway so it probably doesn't matter.